docs/faststore/docs/customization/building-sections/override-use-cases/adding-an-image-to-the-alert-component.mdx

```diff
@@ -19,88 +19,95 @@ By leveraging `getOverriddenSection`, you can create a custom component called `
-
-The following schema was used as an example:
-
-    ```json 
+   ```tsx focus=10:31
+   import { useMemo } from "react";
+   import { AlertSection, getOverriddenSection } from "@faststore/core";
+   import { Image_unstable as Image } from "@faststore/core/experimental";
+
+   interface AlertWithImageProps
+     extends Omit<React.ComponentProps<typeof AlertSection>, "icon"> {
+     src: string;
+     alt: string;
+   }
+
+   export default function AlertWithImage(props: AlertWithImageProps) {
+     const { src, alt, ...otherProps } = props;
+
+     const OverriddenAlert = useMemo(
+       () =>
+         getOverriddenSection({
+           Section: AlertSection,
+           className: styles.alertWithImage,
+           components: {
+             Icon: {
+               Component: () => (
+                 <Image
+                   src={props.src}
+                   alt={props.alt}
+                   width={24}
+                   height={24}
+                 />
+               ),
+             },
+           },
+         }),
+       []
+     );
+
+     return <OverriddenAlert {...otherProps} icon="" />;
+   }
+   ```
+
+   - This function receives the `AlertWithImageProps`.
+   - It destructures the `src` prop containing the image source and remaining props using the spread operator (`...otherProps`).
+   - The core functionality lies within the `useMemo` hook.
+
+     - It defines the `OverriddenAlert` constant using `getOverriddenSection`.
+     - The override options specify:
+       - **`Section`:** The section to be overridden (Alert).
+       - **`components`:** An object containing the override for the Icon component.
+
+   - Inside the Icon override:
+
+     - **`Component`:** A function returning an `Image` component. This displays the image specified by the `src` prop.
+     - **`width`** and **`height`:** Optional props to set the size of the image.
+
+   - The use of `useMemo` ensures this override is only created once, improving performance.
+   - The function returns the `OverriddenAlert` component, spreading the `otherProps` and setting the `icon` prop to `noop` (effectively disabling the default icon behavior).
+
+   > ℹ️ For further details on code implementation, see the [`AlertWithImage`](https://github.com/vtex-sites/playground.store/tree/main/src/components/sections/AlertWithImage) folder available in the [playground.store](https://github.com/vtex-sites/playground.store) repository.
+
+6. Add the section to the Headless CMS by following the instructions available in [syncing components to the Headless CMS](https://developers.vtex.com/docs/guides/faststore/overrides-syncing-components-with-the-headless-cms).
+
+    The following schema was used as an example in [`cms/faststore/sections.json`](https://github.com/vtex-sites/playground.store/blob/main/cms/faststore/sections.json#L58-L101):
+
+    ```json
```

Comment:
please reformat the code inside this code block, some properties are not properly indented. it should be like:

```json
[
  {
    "name": "AlertWithImage",
    "schema": {
      "title": "Alert with image",
      "description": "Add an Alert that has an Image as an icon",
      "type": "object",
      "required": [
        "src",
        "content",
        "dismissible"
      ],
      "properties": {
        "src": {
          "type": "string",
          "title": "Image source",
          "description": "Source of the Alert icon image"
        },
        "content": {
          "type": "string",
          "title": "Content"
        },
        "link": {
          "title": "Link",
          "type": "object",
          "properties": {
            "text": {
              "type": "string",
              "title": "Link Text"
            },
            "to": {
              "type": "string",
              "title": "Action link"
            }
          }
        },
        "dismissible": {
          "type": "boolean",
          "default": false,
          "title": "Is dismissible?"
        }
      }
    }
  }
]
```